app/src/components/IconSvg/WalletSvg/WalletConnect.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export const WalletConnect: React.FC<{ width?: number }> = ({ width = 24 }) => {
```

Comment:
Let the SVGs export default, this makes it possible for us to use lazy dynamic imports with them.